pubtools/_content_gateway/push_base.py

```diff
@@ -314,31 +321,53 @@ def process_product(self, item):
         product_name = item.get('metadata').get('name')
         product_code = item.get('metadata').get('productCode')
         LOG.debug(
-            "Fetching for product_id of product_name:- %s and product_code:- %s" % (product_name, product_code))
+            "Fetching for product_id of product_name:- %s and product_code:- %s \n" % (product_name, product_code)
```

Comment:
Is the \n necessary?

Reply:
Added just for log formatting to look more crumbled.